app/Http/Requests/Admin/StoreUser.php

```diff
@@ -29,7 +29,7 @@ public function rules()
             'last_name' => 'required|string|max:255',
             'organization_name' => 'max:255',
             'job_title' => 'max:255',
-            'email' => 'required|email|max:255|unique:users',
+            'email' => 'required|email|max:255|unique:users,email,NULL,id,deleted_at,NULL', // pass the email if record is soft-deleted issue#CUR-537
```

Comment:
What is this validation?

Reply:
This validates skip the users  if soft deleted. Leo asked me to re-activate the user account if soft-deleted user signs up again.

Reply:
Understand but why do we have two NULLs there?